### ISSUE TITLE
Expo sdk 48 issue TypeError: undefined is not an object (evaluating ‘segments[0][1]’)

### ISSUE THREAD

**Issue by TankPOOJA** [coinjar/react-native-wagmi-charts#133]
![IMG_8853](https://user-images.githubusercontent.com/40266663/235435619-aff88c23-8542-41dd-94c5-89183ffc8e1f.png)

![IMG_8854](https://user-images.githubusercontent.com/40266663/235444765-5afa6f62-98f2-40f3-a03f-3577df09f689.png)

I updated expo sdk, after that app crashed with this error, any solution for this issue.
    "expo": "^48.0.0",
    "react-native": "^0.71.7",
    "react-native-wagmi-charts": "^2.3.0",
    "react-native-svg": "13.4.0",
    "react-native-reanimated": "^2.14.4",
    "react-native-gesture-handler": "^2.5.0",

**Comment by TankPOOJA** [comment]
any solution?

**Comment by arancauchi** [comment]
Thanks for the issue @TankPOOJA, is there any way your could create a basic reproduction repro for this and send it over?
I can't seem to replicate this issue.

**Comment by JiriHoffmann** [comment]
Ran into the same issue. I believe the root cause of it is when `LineChart.Provider` is provided with an empty array for the `data` prop. 

An empty string is passed to `parse` from `react-native-redash` causing it to error out.
https://github.com/coinjar/react-native-wagmi-charts/blob/767ab380271b18fbb44ef8969bc68031d65e589f/src/charts/line/Chart.tsx#L93

**Comment by gerryfletch** [comment]
Yeah, empty `data` prop was the cause for me. Thanks @JiriHoffmann

**Comment by honeybadger26** [comment]
Hi @TankPOOJA. Were you able to resolve this issue by setting a non-empty `data` prop?
